doc/classes/EditorSettings.xml

```diff
@@ -763,6 +766,9 @@
 		<member name="text_editor/behavior/files/restore_scripts_on_load" type="bool" setter="" getter="">
 			If [code]true[/code], reopens scripts that were opened in the last session when the editor is reopened on a given project.
 		</member>
+		<member name="text_editor/behavior/files/select_scene_root_script_on_load" type="bool" setter="" getter="">
+			If [code]true[/code], selects the root node's script in the script editor when opening a scene.
```

Comment:
Isn't this the same as `open_dominant_script_on_scene_change`?

Reply:
Actually yes, somehow I managed to overlook `open_dominant_script_on_scene_change`. Removed the new setting. 
Regarding the setting itself: Personally I think this should be turned off by default. I find it more useful when the last script I worked on is restored. What do you think? Maybe we should make a poll?